lab-carolina/lib/middleware.js

```diff
@@ -0,0 +1,13 @@
+const createError = require('http-errors');
```

Comment:
This is solid, though you probably want to name this file differently since you'll inevitably end up having more than one middleware module.